src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DeliverySessionDTO.kt

```diff
@@ -34,6 +34,8 @@ data class UpdateAppointmentDTO(
   val sessionType: AppointmentSessionType? = null,
   val appointmentDeliveryAddress: AddressDTO? = null,
   val npsOfficeCode: String? = null,
+  val appointmentAttendance: UpdateAppointmentAttendanceDTO? = null,
+  val appointmentBehaviour: RecordAppointmentBehaviourDTO? = null,
 )
```

Comment:
Restarting comment from above - should there be a DeliveryAppointmentDTO also be included?